frontend/src/store/slicers/userSlice.ts

```diff
@@ -0,0 +1,26 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit"
+
+export type User = {
+    email: string,
+    fullName: string
+};
+
+export type InitialState = {
+    user: User | null
+};
```

Comment:
tbh you usually don't need to use semicolon after type declaration but it make sense to use semicolon after each field in the declarationn like:
```
export type InitialState = {
    user: User | null;
}
```